### PR TITLE
Legend rework and Geography filter removal

### PR DESCRIPTION
**Pull request by brett-stephen**
## Briefly describe the feature or bug that this PR addresses.
- Adds toggles to each individual pin type in the legend
- Removes estimate grade from filter
## Please link the Airtable ticket associated with this PR.
https://airtable.com/tbli2lWQHAqBa6ZcI/viw8Ro4zYPYcBGXRw/recHMdR15tgClYpiG?blocks=hide

## If applicable, include screenshots of the feature/bugfix introduced by this PR (Please include both desktop and mobile if there is a significant UI change).
image
## Describe the steps you took to test the feature/bugfix introduced by this PR.
- Some quick smoke testing to verify the filters work correctly

## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.
- may need to deprecate the filters for estimate grade

### CODE REVIEW

**Comment by atatmaja** [comment]
Code itself looks good, but are we sure it's better to do use a "estimate grade" layer instead of just making the legend toggles work like filters? I guess the pros and cons of this approach that I have in mind are:

- Pros: May achieve faster performance on toggle 
- Cons: Less code reuse / more to maintain, if intended behaviour around the estimate grade filter/toggle ever changes we would have to change in multiple places instead of just making a change on the backend

File: src/context.tsx [hunk 1]
```diff
@@ -164,9 +190,5 @@ const reducer = (state: State, action: Record<string, any>): State => {
 export const AppContextProvider = (props: Record<string, any>) => {
   const [state, dispatch] = useReducer(reducer, initialState);
 
-  return (
-    <AppContext.Provider value={[state, dispatch]}>
-      {props.children}
-    </AppContext.Provider>
-  );
+  return <AppContext.Provider value={[state, dispatch]}>{props.children}</AppContext.Provider>;
```
Comment (atatmaja) [inline src/context.tsx:193]: did your linter do this? looks kinda ugly tbh lmao

File: src/components/map/Legend.tsx [hunk 2]
```diff
@@ -9,40 +9,50 @@ import { MapSymbology } from "components/map/MapConfig"
...
+        <div className="legend-item" id="Local" onClick={handleLegendToggle}>
           <i className="circleBase" style={{ background: MapSymbology.StudyFeature.Local.Color }}></i>
-          <p className={isMobileDeviceOrTablet ? "mobile-text px-2" : "px-1"}>{Translate("LocalStudies")}</p>
+          <p>{Translate("LocalStudies")}</p>
+          <input className="checkbox" type="checkbox" checked={state.explore.legendLayers.Local} readOnly/>
         </div>
         <div className="legend-item">
           <i className="block" style={{ background: MapSymbology.CountryFeature.HasData.Color, outlineWidth: 1, outlineStyle: "solid" }}></i>
-          <p className={isMobileDeviceOrTablet ? "mobile-text px-2" : "px-1"}>{Translate("CountryEstimatesExist")}</p>
+          <p>{Translate("CountryEstimatesExist")}</p>
```
Comment (atatmaja) [inline src/components/map/Legend.tsx:47]: if we're no longer applying conditional styling on mobile vs desktop, did we test to see how it looks on both screen sizes?

**Comment by brett-stephen** [comment]
image

Updated Legend to be more compact & use proper semantic ui check boxes

File: src/components/map/Legend.tsx (hunk 2 above)
Comment (brett-stephen) [reply src/components/map/Legend.tsx:46]: Legend is compact enough where we don't need to change styling for mobile. This version takes less space and looks better than the prev mobile legend
> 
> 
> image
> 
> Updated Legend to be more compact & use proper semantic ui check boxes

**Comment by atatmaja** [comment]
Can u share screenshot of what it looks like on mobile?

**Comment by atatmaja** [comment]
Not necessary for this PR but would be cool if we could make the legend collapsable (for power users who are familiar with the legend and want more screen real estate) @brett-stephen @jordanvanwyk what do you think

**Review by atatmaja — APPROVED** [review]
Code LGTM, can merge once we agree that mobile version looks ok

**Comment by jordanvanwyk** [comment]
maybe we can fit legend into the side bar to fit with tableau as another option. we can do more mocks 
https://www.figma.com/file/duS85QuRG7WPz6AlPwEmBk/Legend?node-id=0%3A1 <https://www.figma.com/file/duS85QuRG7WPz6AlPwEmBk/Legend?node-id=0:1>

**Comment by brett-stephen** [comment]
We could make the legend have the same behavior as filters and map statistics on mobile. I'm not sure if being able to hide the legend would be an issue for the WHO

**Comment by brett-stephen** [comment]
Moved mobile legend into a sidebar button 
State - Non expanded:
image
State - Expanded: 
image

**Comment by brett-stephen** [comment]
Removed legend toggle, and cleaned up & adjusted the CSS.
Desktop:
image

Mobile:
image

**Pull request merged**